opm/parser/eclipse/Parser/tests/MessageContainerTest.cpp

```diff
@@ -34,8 +34,8 @@ BOOST_AUTO_TEST_CASE(TestIterator) {
     msgContainer.bug("This is a bug.", "dummy.log", 20);
     {       
         BOOST_CHECK_EQUAL("This is an error.", msgContainer.begin()->message);
-        BOOST_CHECK_EQUAL("dummy.log", (msgContainer.end()-1)->location->filename);
-        BOOST_CHECK_EQUAL(20, (msgContainer.end()-1)->location->lineno);
+        BOOST_CHECK_EQUAL("dummy.log", (msgContainer.end()-1)->location.filename);
+        BOOST_CHECK_EQUAL(20, (msgContainer.end()-1)->location.lineno);
     }
```

Comment:
Can we have a test demonstrating the `operator bool( )` functionality of the `Location` class.